### ISSUE TITLE
Portfolio percentage

### ISSUE THREAD

**Issue by hajo62** [gwendo/entities-table-card#8]
Not knowing if this is possible...

Calculate the % profit/loss for the summarized profit/loss in relation to the summarized value.
In this case: 3003/(7004-3003)  =  75.1%
<img width="481" alt="grafik" src="https://user-images.githubusercontent.com/15800980/111196062-30dc7700-85bd-11eb-9c63-d4c8905e31a8.png">

**Comment by gwendo** [comment]
@hajo62 What you are talking about is some kind of dsl to specify how the summary row should be calculated for each column, am I understanding you correctly then?

**Comment by hajo62** [comment]
Not sure to understand.
The last two colums calculate a sum by default. Would be great to have that for %-column as well.
But because it's %, it's to be weighted by the stocks you own. 
Or calculated viceversa using the sums:
100 * Profit/Loss / (Acutal - Profit/Loss) = 75,1%

**Comment by gwendo** [comment]
Yes, in your specific case it would work, however to keep this table card generic I can't assume that those columns are available.
That's why you would need some kind of expression language with support for defining those kinds of summaries.
